src/client/components/forms/relationship.js

```diff
@@ -111,6 +115,7 @@ class RelationshipEditor extends React.Component {
 	/**
 	 * Submits the changed, valid relationships.
 	 * It is triggered by the onClick event of the SUBMIT button.
+	 * @returns {void}
```

Comment:
No need for this - sorry, the rules for documentation are quite new, and this is one of the things we're planning to change - for now, please add `/* eslint valid-jsdoc: ["error", { "requireReturn": false }] */` to the top of the file (below the copyright header and license), then remove the `@returns {void}` lines from the file.